### ISSUE TITLE
2.37 fails to build on CentOS7

### ISSUE THREAD

**Issue by hahnjo** [karelzak/util-linux#1405]
GCC 4.8.5 complains
```
error: expected ';', ',' or ')' before 'fmt'
 static void uuid_fmt(const uuid_t uuid, char *buf, char const *restrict fmt)
```

My superficial analysis is that `restrict` is a C99 keyword and as `util-linux` doesn't specify a `-std=` argument, the compiler uses its default which happens to be `c89` on CentOS7? Passing `CFLAGS=-std=c99` to `configure` makes it work again.

**Comment by karelzak** [comment]
We already have some code in c.h to make `restrict` more portable, but this header file has not been included in libuuid. The problem should be fixed now.

**Comment by hahnjo** [comment]
Thanks for the super-fast reply and fix, I can confirm this fixes the build :+1:

**Comment by hgy59** [comment]
unfortunately the fix (commit 5f9b88f43ba7f98f81bde3538d5f4e5cd1a6c01c) is not contained in the current release v2.37.2 (missing in stable/v2.37 branch).

**Comment by karelzak** [comment]
> unfortunately the fix (commit [5f9b88f](https://github.com/util-linux/util-linux/commit/5f9b88f43ba7f98f81bde3538d5f4e5cd1a6c01c)) is not contained in the current release v2.37.2 (missing in stable/v2.37 branch).

My plan is to release v2.38 and v2.37.3 at the end of January.
